sites/skeleton.dev/src/content/docs/framework-components/tabs.mdx

```diff
@@ -114,6 +114,78 @@ Set the text direction (`ltr` or `rtl`) using the `dir` prop.
 	</Preview>
 </Framework>
 
+## Route Controlled
+
+When using frameworks like SvelteKit and React Router, controlling the active tabs based on the URL can be useful.
+
+To achieve this, you need to do two things:
+
+- Set the `value` prop to the current URL path.
+- Listen to the `navigate` event and update the URL path.
+
+<Framework id="react">
+
+```tsx
+import { Tabs } from '@skeletonlabs/skeleton-react';
+import { Outlet, useNavigate } from 'react-router';
```

Comment:
This is a question for @Hugos68  - are we good with the use of React Router here? Personally I'd recommend we lean into whatever Next.js recommends. Which may be this?

Also I'd appreciate a once-over of the React code as Jens mentioned this is new for him.